From 6c9d0918abff1161d905e9dec5616ae558357d4b Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Tue, 3 Mar 2015 12:58:22 +0100
Subject: [PATCH] connection-editor,vlan: don't disable the system connection checkbox if the connection is not valid

It's confusing. No reason to do it anyway, "Apply" button is still disabled.
---
 .../src/connection-editor/nm-connection-editor.c   |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/network-manager-applet-0.8.1/src/connection-editor/nm-connection-editor.c b/network-manager-applet-0.8.1/src/connection-editor/nm-connection-editor.c
index bedb18f..1543ac7 100644
--- a/network-manager-applet-0.8.1/src/connection-editor/nm-connection-editor.c
+++ b/network-manager-applet-0.8.1/src/connection-editor/nm-connection-editor.c
@@ -200,8 +200,7 @@ update_sensitivity (NMConnectionEditor *editor)
 		/* Bond and bridge connections and their slaves must be system connections */
 		gtk_widget_set_sensitive (GTK_WIDGET (editor->system_checkbutton), FALSE);
 	} else {
-		gtk_widget_set_sensitive (GTK_WIDGET (editor->system_checkbutton),
-		                          actionable && authorized);
+		gtk_widget_set_sensitive (GTK_WIDGET (editor->system_checkbutton), authorized);
 	}
 
 	/* Cancel button is always sensitive */
-- 
1.7.1

